frontend/src/components/software/SoftwareDetailsContainer.jsx

```diff
@@ -1,10 +1,107 @@
 import React, {Component} from 'react';
 import SoftwareDetailsComponent from './SoftwareDetailsComponent';
+import isEmpty from "react-native-web/dist/vendor/react-native/isEmpty";
```

Comment:
Nie importujmy nic z `react-native-web`, gdyż to nie zadziała na apce mobilnej

Reply:
wyrzucone, zastąpiłem to warunkiem x === ''